github/libp2p/membership.json

```diff
@@ -176,42 +104,21 @@
   "jbenet": {
     "role": "admin"
   },
-  "jbenetsafer": {
-    "role": "member"
-  },
   "jchris": {
     "role": "member"
   },
   "jrhea": {
```

Comment:
They've been involved in this category of actions this may - https://docs.github.com/en/enterprise-cloud@latest/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization#org_credential_authorization-category-actions

I don't think it's relevant. Let me update the script to stop considering these kind of actions altogether.